diff --git a/notflix b/notflix
index f10c051..a7a1c60 100755
--- a/notflix
+++ b/notflix
@@ -4,12 +4,12 @@ LANG=en_IN.utf-8
 
 mkdir -p $HOME/.cache/notflix
 
-menu="dmenu -i -h 30 -l 25"
+menu="dmenu -i -l 25"
 baseurl="https://1337x.wtf"
 cachedir="$HOME/.cache/notflix"
 
 if [ -z $1 ]; then
-  query=$(dmenu -p "Search Torrent: " -h 30 <&-)
+  query=$(dmenu -p "Search Torrent: "<&-)
 else
   query=$1
 fi
@@ -67,7 +67,7 @@ fullURL="${baseurl}${url}/"
 # Requesting page for magnet link
 curl -s $fullURL > $cachedir/tmp.html
 magnet=$(grep -Po "magnet:\?xt=urn:btih:[a-zA-Z0-9]*" $cachedir/tmp.html | head -n 1) 
-player=$(echo -e "vlc\nmpv"|dmenu -i -p "Player:" -h 30 )
+player=$(echo -e "vlc\nmpv"|dmenu -i -p "Player:" )
 cmd="webtorrent "$magnet" --$player"
 (!(alacritty msg create-window --command $cmd))&& alacritty --command $cmd
 rm -rf /tmp/webtorrent/*
